runelite-client/src/main/java/net/runelite/client/plugins/devtools/WidgetInspector.java

```diff
@@ -335,9 +334,21 @@ static String getWidgetName(int componentId)
 			//until it's actually needed.
 			try
 			{
-				for (Field f : ComponentID.class.getDeclaredFields())
+				for (Field field : InterfaceID.class.getDeclaredFields())  // is this block unnecessary?
```

Comment:
This would assign the interface ids to any component with id 0 which is not totally correct I think.